src/archetype/createProject/SelectArchetypeStep.ts

```diff
@@ -62,31 +74,15 @@ export class SelectArchetypeStep implements IProjectCreationStep {
         }
     }
 
-    public async run_simple(metadata: IProjectCreationMetadata): Promise<StepResult> {
```

Comment:
@cscherryy I find this method unused. Is it safe to remove?

Reply:
looks like the method is introduced by https://github.com/microsoft/vscode-maven/pull/619/files#diff-91272aa6b61f071f70239054717cefd20b164ca21c2a227d03c00195a514814e.

Anyway, IMO we can remove it.